apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -101,6 +106,15 @@ export function TokenDetails({ coinType }: TokenDetailsProps) {
         refetchInterval: DELEGATED_STAKES_QUERY_REFETCH_INTERVAL,
     });
 
+    const { data: supplyIncreaseVestingObjects } = useGetAllOwnedObjects(
+        activeAccountAddress || '',
+        {
+            StructType: TIMELOCK_IOTA_TYPE,
```

Comment:
We are only looking at timelocked objects, but we should also look at timelocked objects stacked with `useGetTimelockedStakedObjects`